Toolset/palettes/script editor/behaviors/revsecommoneditorbehavior.livecodescript

```diff
@@ -2212,6 +2232,10 @@ on rawKeyDown pKey
          put "right" into tKeyName
          break
       case kKeyUpArrow
+         # 2017-JUL-28 bhall2001
+         # when scrolling on up arrow, scrollBarDrag is sent twice. Set
+         # flag to update on every other up arrow message
+         put not sSkipUpArrow into sKeyUpArrow
```

Comment:
@bhall2001 "put not sSkipUpArrow into sKeyUpArrow" sKeyUpArrow should probably be "sSkipUpArrow"